packages/mdc-list/mdc-list.scss

```diff
@@ -195,12 +198,19 @@ a.mdc-list-item {
   border-bottom-style: solid;
 }
 
+.mdc-list-divider--padded {
```

Comment:
Nit: to me, the word `padded` in the class name implies `padding`, but the implementation actually uses `margin` to achieve the effect.

Do you think that difference might confuse or surprise people?

Reply:
Can we think of a better word to use? Usually I'm against using terms that can seem contradictory in the source language, but I'm not sure what else works here. `inset` is already used for a different purpose.

Reply:
It also occurs to me that although in this particular case `margin` is used, the purpose of the `--padded` modifier is to match the `padding` that is applied to individual list items. We need to use `margin` here since the intent here is to restrict the border.

Reply:
I see - let's keep `--padded` then.